Makefile

```diff
@@ -108,6 +108,9 @@ build-daemon:
 run: manifests generate fmt vet ## Run a controller from your host.
 	go run cmd/operator/main.go
 
+ifndef IGNORE-NOT-FOUND
```

Comment:
I feel the name of the variable lacks of context in the scope of the script; consider `IGNORE_NOT_FOUND_ON_DELETION`

Reply:
I was going to change it to this, but I decided against it since this is OSDK generated code.